packages/web-components/fast-components/src/accordion-item/index.ts

```diff
@@ -6,13 +6,13 @@ import {
 import { accordionItemStyles as styles } from "./accordion-item.styles";
 
 /**
- * The FAST Accordion Item Element. Implements {@link @microsoft/fast-foundation#AccordionItem},
- * {@link @microsoft/fast-foundation#accordionItemTemplate}
+ * A function that returns a {@link @microsoft/fast-foundation#AccordionItem} registration for configuring the component with a DesignSystem.
```

Comment:
Is there another link we can add for the `DesignSystem`?

Reply:
Good question - @nicholasrice or @EisenbergEffect, would `{@link @microsoft/fast-foundation#DesignSystem}` be accurate here? I can update all.

Reply:
yep, that's the link